bats/tests/helpers/paths.bash

```diff
@@ -48,6 +48,13 @@ set_path_resources() {
     if [ ! -d "$PATH_RESOURCES" ]; then
         fatal "App resource directory '$PATH_RESOURCES' does not exist"
     fi
+
+    # Ensure we're bundled executables for sub-processes
+    # (e.g. credential helpers).
```

Comment:
I suspect the comment is missing a "using" in there.

It is also only about credential helpers, afaict, because all other utilities are explicitly called with a full path from the wrappers in `helpers/commands.bash`.

Reply:
I guess theoretically we could just temporarily update the `PATH` in the `ctrctl` wrapper, but it doesn't hurt to do this globally.